pkg/model/api/v1/dashboard.go

```diff
@@ -129,7 +130,8 @@ func (d *DashboardSpec) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 func (d *DashboardSpec) validate() error {
 	if len(d.Panels) == 0 {
-		return fmt.Errorf("spec.panels cannot be empty")
+		logrus.Trace("the dashboard panels list is empty")
```

Comment:
Personally I'm fine to no longer raise a validation error when having an empty panels list, WDYT @Nexucis? Alternatively the solution for the grafana migration here would be (as described in https://github.com/perses/perses/issues/1536) "_add some logic to append a dummy panel when migrating a Grafana dashboard that doesn't embed any._"

Reply:
well on the functional side, I find it weird to be able to create an empty dashboard. But at the end probably it might simplify other flow such as the creation of the dashboard. Maybe something to be done in another PR

Reply:
but we shouldn't use a logger in this package. That's not the correct place for that.
In this package you mostly have the definition of the different resource type used in Perses. This package is supposed to be "free" to import. That means you won't have any external definition.
The only exception we did was about the type `duration` coming from Prometheus.

Can you remove the log @ying-jeanne please ? And then I believe we should be good to merge it.
Thank you for this nice contribution !!

Reply:
hi folks, logline is removed.